### PR TITLE
Step1 bahrom

### PR DESCRIPTION
**Pull request by BahaMaksudov**
Updated ProductStatus Enum - changed Passive to Inactive

### CODE REVIEW

**Review by AbbosB12 — APPROVED** [review]

File: src/main/java/com/example/accountingapp/enums/ProductStatus.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package com.example.accountingapp.enums;
 
 public enum ProductStatus {
-    ACTIVE("Active"), PENDING("Pending");
```
Comment (AccountingApp-Group2) [inline src/main/java/com/example/accountingapp/enums/ProductStatus.java:4]: test

File: src/main/resources/data.sql [hunk 2]
```diff
@@ -11,7 +11,7 @@ VALUES (1, '2022-01-05 00:00:00', FALSE, 1, '2022-01-05 00:00:00', TRUE, '04/15/
 
 INSERT INTO product (created_by, created_time, is_deleted, updated_by, updated_time, description, enabled, low_limit_alert, name, new_column, product_status, qty, tax, unit)
 VALUES (1, '2022-04-15 00:00:00', FALSE, 1, '2022-05-07 00:00:00', 'computer', TRUE, 15, 'MacBook', 1, 'ACTIVE', 15, 4, 'Pieces'),
-       (2, '2022-03-17 00:00:00', FALSE, 2, '2022-05-20 00:00:00', 'sports', TRUE, 7, 'Shoe', 1, 'ACTIVE', 7, 2, 'Pieces'),
```
Comment (AccountingApp-Group2) [inline src/main/resources/data.sql:14]: test

File: src/main/java/com/example/accountingapp/enums/ProductStatus.java (hunk 1 above)
Comment (AccountingApp-Group2) [reply src/main/java/com/example/accountingapp/enums/ProductStatus.java:4]: ok

**Review by AccountingApp-Group2 — APPROVED** [review]

File: src/main/resources/data.sql (hunk 2 above)
Comment (BahaMaksudov) [reply src/main/resources/data.sql:14]: Thanks!

**Pull request merged**